tcl/response.tcl

```diff
@@ -55,21 +55,23 @@ namespace eval qc::response {
         namespace export invalid valid remove all_valid
         namespace ensemble create
 
-        proc valid {name value {message ""}} {
+        proc valid {name value {message ""} {sensitive false}} {
             #| Adds the given field to the record as valid. If the field already exists then updates it.
             global data
             dict set data record $name valid true
             dict set data record $name value $value
             dict set data record $name message $message
+            dict set data record $name sensitive $sensitive
         }
 
-        proc invalid {name value message} {
+        proc invalid {name value message {sensitive false}} {
             #| Adds the given field to the record as invalid. If the field already exists then updates it.
             #| Also sets the status of the response to invalid.
             global data
             dict set data record $name valid false
             dict set data record $name value $value
-            dict set data record $name message $message
+            dict set data record $name message $message            
+            dict set data record $name sensitive $sensitive
             qc::response status invalid
         }
```

Comment:
if validate2model flags a field as sensitive by doing:
```
qc::response record valid password secret "Password Valid" true
```

what if there were custom validation handlers like something to check password complexity and they did:
```
qc::response record invalid password secret "Password not complex enough"
```

The field would no longer be flagged as sensitive.
======================================

Of course the developer could have done the following (but he might not remember):
```
qc::response record invalid password secret "Password not complex enough" true
```

or
 ```
qc::response record invalid password "" "Password not complex enough"
```